mohawk/tests.py

```diff
@@ -637,7 +637,16 @@ def test_receive_wrong_content(self):
     @raises(MisComputedContentHash)
     def test_unexpected_unhashed_content(self):
         self.receive(sender_kw=dict(content=EmptyValue,
-                                    content_type=EmptyValue,
+                                    content_type='text/plain',
+                                    always_hash_content=False))
+
+    def test_expected_unhashed_empty_content(self):
+        content = EmptyValue
```

Comment:
When `content` is `EmptyValue`, `MissingContent` should still be raised. In other words, you should keep this test but add `@raises(MissingContent)` to the top. After that, add two more tests. One where `content=''` and another where `content=None`.